### PR TITLE
chore: new stable release

### PR DESCRIPTION
**Pull request by balazsorban44**
**What**:

**Why**:

**How**:

**Checklist**:

- [ ] Documentation
- [ ] Tests
- [ ] Ready to be merged

### CODE REVIEW

**Comment by balazsorban44** [comment]
Just a note. This should NOT be squashed and merged!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+Thanks for your interest in the project. Bugs filed and PRs submitted are appreciated!
+
+Please make sure that you are familiar with and follow the Code of Conduct for
+this project (found in the CODE_OF_CONDUCT.md file).
+
+Also, please make sure you're familiar with and follow the instructions in the
+contributing guidelines (found in the CONTRIBUTING.md file).
+
+If you're new to contributing to open source projects, you might find this free
+video course helpful: https://kcd.im/pull-request
```
Comment (prisis) [inline .github/PULL_REQUEST_TEMPLATE.md:11]: The url is not working

Comment (balazsorban44) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: Thanks! I will ask @kentcdodds if he could fix the redirect.

kcd.im/pull-request should point to:
```diff
-https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github?af=5
+https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github?af=5
```

Comment (kentcdodds) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: This has been fixed. Thanks for the tip!

**Review by ndom91 — APPROVED** [review]
LGTM :+1:

**Pull request merged**